metron-deployment/roles/kafka-broker/tasks/main.yml

```diff
@@ -1,41 +0,0 @@
-#
-#  Licensed to the Apache Software Foundation (ASF) under one or more
-#  contributor license agreements.  See the NOTICE file distributed with
-#  this work for additional information regarding copyright ownership.
-#  The ASF licenses this file to You under the Apache License, Version 2.0
-#  (the "License"); you may not use this file except in compliance with
-#  the License.  You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-#
----
-- name: Retrieve HDP repository definition
-  get_url:
-    url: "{{ hdp_repo_def }}"
-    dest: /etc/yum.repos.d/hdp.repo
-    mode: 0644
-
-- name: Install kafka
-  yum: name={{item}}
-  with_items:
-    - java-1.8.0-openjdk
-    - kafka
-    - zookeeper-server
-
-- name: Create pid directories
-  file: path={{ item }} state=directory mode=0755
-  with_items:
-    - /var/run/zookeeper
-    - /var/run/kafka
-
-- name: Start zookeeper
-  shell: /usr/hdp/current/zookeeper-server/bin/zookeeper-server start
-
-- name: Start kafka
```

Comment:
The `kafka_broker` role is used by the `fastcapa-test-platform`.  Removing it will break that environment.